packages/datadog-plugin-mongodb-core/test/mongodb.spec.js

```diff
@@ -614,6 +614,55 @@ describe('Plugin', () => {
         })
       })
 
+      describe('with dbmPropagationMode full but sampling disabled', () => {
+        before(() => {
+          tracer._tracer.configure({ env: 'tester', sampler: { sampleRate: 0 } })
+
+          return agent.load('mongodb-core', {
+            dbmPropagationMode: 'full'
+          })
+        })
+
+        after(() => {
+          tracer._tracer.configure({ env: 'tester', sampler: { sampleRate: 1 } })
+
+          return agent.close({ ritmReset: false })
+        })
+
+        beforeEach(async () => {
+          client = await createClient()
+          db = client.db('test')
+          collection = db.collection(collectionName)
+
+          startSpy = sinon.spy(MongodbCorePlugin.prototype, 'start')
```

Comment:
This spy is not needed, only the behaviour needs to be tested.

Reply:
Never mind, I just saw that there are other occurrences in this file, so fixing this is out of scope of this PR then.